src/main/java/org/dependencytrack/event/kafka/processor/VulnerabilityScanResultProcessor.java

```diff
@@ -68,9 +114,14 @@ public void process(final FixedKeyRecord<ScanKey, ScanResult> record) {
         final UUID componentUuid = UUID.fromString(scanKey.getComponentUuid());
         final VulnerabilityAnalysisLevel analysisLevel = determineAnalysisLevel(record);
         final boolean isNewComponent = determineIsComponentNew(record);
+
         final Timer.Sample timerSample = Timer.start();
-        try (final var qm = new QueryManager().withL2CacheDisabled()) {
-            final Component component = qm.getObjectByUuid(Component.class, componentUuid, List.of(Component.FetchGroup.IDENTITY.name()));
+        try (final var qm = new QueryManager()) {
+            // Do not unload fields upon commit (why is this even the default WTF).
```

Comment:
the comment on code :P

Reply:
Well what can I say, that default behavior annoys me to no end :D